cluster/CHAPTER.rst

```diff
@@ -170,8 +170,7 @@ In concrete terms, this means adding a slot number to each message, with all of
 Running Paxos for every slot, with its minimum of two round trips, would be too slow.
 Multi-Paxos optimizes by using the same set of ballot numbers for all slots, and performing the ``Prepare``/``Promise`` phase for all slots at once.
 
-Paxos Made .. Pretty Hard, Actually
------------------------------------
+### Paxos Made… Pretty Hard, Actually
```

Comment:
@MichaelDiBernardo I put an ellipsis in here -- I'm not sure how that will be encoded in LaTeX or epub, so let's keep an eye on it.